systemd-units/meson.build

```diff
@@ -7,6 +7,11 @@ install_data(
   install_dir : systemd_unit_dir
 )
 
+install_data(
```

Comment:
I think you need to add this file to the rpm build below [the hirte files](https://github.com/containers/hirte/blob/main/hirte.spec.in#L85)

Reply:
The PR includes a change in hirte.spec.in. Isn't this what you are referring to?

Reply:
Oh, missed that sneaky file. You are right. Sorry about that.